policy_set.go

```diff
@@ -115,6 +120,18 @@ type PolicySetCreateOptions struct {
 	// Whether or not the policy set is global.
 	Global *bool `jsonapi:"attr,global,omitempty"`
 
+	// The sub-path within the attached VCS repository to ingress. All
+	// files and directories outside of this sub-path will be ignored.
+	// This option may only be specified when a VCS repo is present.
+	PoliciesPath *string `jsonapi:"attr,policies-path,omitempty"`
+
+	// VCS repository information. When present, the policies and
+	// configuration will be sourced from the specified VCS repository
+	// instead of being defined within the policy set itself. Note that
+	// this option is mutually exclusive with the Policies option and
+	// both cannot be used at the same time.
+	VCSRepo *VCSRepoOptions `jsonapi:"attr,vcs-repo,omitempty"`
```

Comment:
Same thing about the ordering and are you sure you need the full `VCSRepoOptions` object here?

Reply:
Yeah, I double-checked on this. We are using the same bits on the backend here, so in essentiality, it's the same type. All of the options are available to the policy set request as seen in [the documentation](https://github.com/hashicorp/terraform-website/blob/39f0af766d655f5b938696371d4df0c232756b79/content/source/docs/enterprise/api/policy-sets.html.md#request-body). I was thinking of spinning off the VCS repo stuff into its own file, but it doesn't necessarily seem to make sense at this time as it's such a minor amount of code.

Reply:
> I was thinking of spinning off the VCS repo stuff into its own file, but it doesn't necessarily seem to make sense at this time as it's such a minor amount of code

Fully agree 👍